### PR TITLE
Update docs subgraph V2

### PR DESCRIPTION
**Pull request by clemsos**
Really just a change in syntax in the way the graph is called.

How do you think we should mention that we have a v1 / v2 breaking changes? @julien51 @searchableguy @wonderwomancode ?

### CODE REVIEW

**Comment by julien51** [comment]
Hum, can you make sure this page https://docs.unlock-protocol.com/tools/subgraph is up to date?

**Comment by clemsos** [comment]
> Hum, can you make sure this page https://docs.unlock-protocol.com/tools/subgraph is up to date?

yep, added. Also I added a notice about the V2 on the same page

File: docs/tools/subgraph.md [hunk 1]
```diff
@@ -13,22 +13,24 @@ Unlock leverages the [Graph Protocol](https://thegraph.com) to simplify the proc
 
 The Graph Protocol provides an accessible GraphQL service providing useful aggregate Unlock data from the Ethereum blockchain. The links below provide a rudimentary tool to build and test your queries.
 
-[Ethereum Mainnet](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock)
+[Ethereum Mainnet](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-v2)
 
-[Rinkeby (Ethereum testnet)](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-rinkeby)
+[Rinkeby (Ethereum testnet)](https://thegraph.com/explorer/subgraph/unlock-protocol/unlock-rinkeby-v2)
```
Comment (julien51) [inline docs/tools/subgraph.md:18]:
```suggestion
```

**Comment by julien51** [comment]
I thought the subgraphs addresses were autogenerated from the network package?

**Comment by julien51** [comment]
I think we are missing Avalanche :/

**Comment by clemsos** [comment]
> I think we are missing Avalanche :/

ok added

**Comment by clemsos** [comment]
> I thought the subgraphs addresses were autogenerated from the network package?

nope just the networks page so far

**Review by julien51 — APPROVED** [review]

**Pull request merged**